src/openassetio-python/CMakeLists.txt

```diff
@@ -99,28 +99,68 @@ if (OPENASSETIO_ENABLE_TESTS AND Python_Interpreter_FOUND)
         COMMAND
         "${Python_EXECUTABLE}" -m venv "${CMAKE_INSTALL_PREFIX}"
         COMMAND
-        ${venv_python_dir}/python -m pip install --upgrade pip
+        ${venv_python_dir}/python -m pip install --upgrade --force-reinstall pip
     )
     # Target to wrap the venv command so it can be executed in isolation.
     add_custom_target(
         openassetio-python-venv
         DEPENDS "${installed_pip}"
     )
 
+    # One-time step to download test-specific dependencies
+    # so we don't always need an online connection for running ctest
+    set(deps_dir "${CMAKE_BINARY_DIR}/dependencies")
+    set(python_deps_dir "${deps_dir}/python")
+    set(test_deps_dir "${python_deps_dir}/tests")
+    # Store signal files under a common directory
+    set(signals_dir "${CMAKE_BINARY_DIR}/signals")
```

Comment:
[consider][minor] Apologies for the churn, but would it be neater to put the signal files in the `dependencies` directory, rather than a separate directory?

Reply:
Addressed in https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/309/commits/77d7dd99809c24f57d8bf11163d99a1e710ff983